argo-cd-apps/base/member/infra-deployments/monitoring-workload-grafana/monitoring-workload-grafana.yaml

```diff
@@ -4,26 +4,38 @@ metadata:
   name: monitoring-workload-grafana
 spec:
   generators:
-    - clusters: {}
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
+              values:
+                sourceRoot: components/monitoring/grafana
+                environment: staging
+                clusterDir: ""
+          - list:
+              elements: []
   template:
     metadata:
       name: monitoring-workload-grafana-{{nameNormalized}}
     spec:
       project: default
       source:
-        path: components/monitoring/grafana/base
+        path: '{{values.sourceRoot}}/{{values.environment}}/{{values.clusterDir}}'
         repoURL: https://github.com/redhat-appstudio/infra-deployments.git
         targetRevision: main
       destination:
-        namespace: appstudio-workload-monitoring
+        namespace: default
```

Comment:
@avi-biton can you add the correct namespace here?

Reply:
I think that we should use `appstudio-grafana` (it will require an update in the other files as well).

Reply:
Done, all Grafana resources are using the `appstudio-grafana` namespace